about.html

```diff
@@ -0,0 +1,144 @@
+
+                    <ul class="nav-ul">
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="about.html">About</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>  
+        </div>
+        
+    </header>
+    <nav>
+        <article class="intro">
+          <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+          <div class="mobileMenu"> 
+            <div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+            <span class="closeMenu">&times;</span>
+              <ul class="container">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>News</li>
+             </ul>
+            </div>
+
+        </article>
+    </nav>
+    <main class="pg1">
+       <img src="./images/about-bg.png" alt="#" class="bckg1">
+            <h2 class="theme">
+                "Youth For Christ"
+             </h2>
+             <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+            <div class="moment">
+                 <p class="prg">the conference is to bring together all youth in all church in the nation under one canmopy to worship God.</p>
+            </div>
+            <div class="contact">
+                <p>please contact us per email for any further questions about winners chapel youth conference! </p>
+                <a  href="">winnerconf@gmail.com</a>
+            </div>
+        </main>
+    
+        <div>
+           <div class="logo1">
+
+                <h3> winners chapel youth conference 2022 logo</h3>
+                <p> winners chapel youth conference 2022 logo ws decided by the youth president bhimself alongside some executives</p>
+            
+            <div class="logo-p">
+                <a href=""><img src="./images/download.jpg" alt="" class="logo-ab"></a>
+            </div>
+        </div>
+        <hr class="line"> 
+    </div>
+<section>
+        <h3 class="abh2">See The Past National Youth Conference</h3>
+       <hr class="hr-f"> 
+        <p class="abp">Take a look at the past two conference which took place in abuja, nigeria</p>
+  <section class="di">    
+<div class="card">
+    <div class="image">
+        <div class="bckg">
+        </div>   
+    </div>
+    <div class="text" >
+    <p class="da">2021 </p>
+<p> youth national conference in lagos</p>
+</div>
+</div>
```

Comment:
- [ ] As the previous reviewer suggested, please align these elements properly so they are centered on the page like on the design.  (I'll help!).

##  Explanation

This is working for you and not for others because you are using `margin-left: %` instead of `margin-auto` or `display-flex`.  You can never guess the multiple screen sizes and pixel densities.  That is why you shouldn't use this approach to center your elements.

This is how it shows for me:

![image](https://user-images.githubusercontent.com/57421823/164949767-11f1b118-5d36-48ba-a34b-50d3f632cff7.png)


## Hints

- As pointed out in my previous review comment, to center horizontally, make use of margin: auto.  
- You can also use Flex, along with `justify-content: center` for horizontally, and `align-items: center` for the vertical center.
- Try not to use fixed measurements, like `px`.  


`margin: auto` and `display: flex` will be your best friends.  For a long time!  Learn to use them and everything will be easier! Please, go ahead and use your new knowledge to your advantage! 💪